velox/functions/sparksql/tests/MapTest.cpp

```diff
@@ -211,5 +211,23 @@ TEST_F(MapTest, complexTypes) {
        makeArrayVector<int64_t>({{1, 2}, {4, 6}})},
       arrayMapResult1);
 }
+
+TEST_F(MapTest, resultSize) {
+  auto inputVector1 = makeNullableFlatVector<int64_t>({1, 2, 3});
```

Comment:
a1, a2,... names are anti pattern; try to avoid these

The results of makeRowVector should be stored in a reference variable. A common pattern is:

```
auto data = makeRowVector({
   make....,
   make....,
});
```